resources/js/processes/designer/Assets.vue

```diff
@@ -8,90 +8,237 @@
     <div class="mt-3">
       <b-card-group deck>
         <b-card
+          v-for="(asset, index) in assets.slice(0, 3)"
+          :key="index"
           bg-variant="light"
           class="text-center"
+          @click="toggleButtons(index)"
         >
-          <asset
-            color="#4DA2EB"
-            icon="fas fa-play-circle"
-            asset_name="Processes"
-          />
-        </b-card>
-        <b-card
-          bg-variant="light"
-          class="text-center"
-        >
-          <asset
-            color="#8EB86F"
-            icon="fas fa-file-alt"
-            asset_name="Screens"
-          />
-        </b-card>
-        <b-card
-          bg-variant="light"
-          class="text-center"
-        >
-          <asset
-            color="#F7CF5D"
-            icon="fas fa-code"
-            asset_name="Scripts"
-          />
+          <div v-show="!showButtons[index]">
+            <asset
+              :color="asset.color"
+              :icon="asset.icon"
+              :asset_name="asset.asset_name"
+            />
+          </div>
+
+          <div
+            v-show="showButtons[index]"
+            class="text-left"
+          >
+            <asset-buttons
+              :asset_name_all="asset.asset_name_all"
+              :variant="variant"
+              :urlPath="asset.urlPath"
+            >
+              <template v-slot:select-template-slot>
+                <div v-if="asset.flag === 1">
+                  <select-template-modal
+                    :type="$t('Process')"
+                    ref="selectTemplateModal"
+                    :countCategories="countCategories"
+                  >
+                  </select-template-modal>
+                </div>
+                <div v-if="asset.flag === 2">
+                  <create-screen-modal
+                    :type="$t('Screens')"
+                    ref="create-screen-modal"
+                    :count-categories="2"
+                    :script-executors="false"
+                    is-projects-installed="false"
+                  />
+                </div>
+                <div v-if="asset.flag === 3">
+                  <create-script-modal
+                    :type="$t('Screen')"
+                    ref="create-script-modal"
+                    :count-categories="2"
+                    :script-executors="false"
+                    is-projects-installed="false"
+                  />
+                </div>
+              </template>
+            </asset-buttons>
+          </div>
         </b-card>
       </b-card-group>
     </div>
     <div class="mt-3">
       <b-card-group deck>
         <b-card
+          v-for="(asset, index) in assets.slice(3)"
+          :key="index"
           bg-variant="light"
           class="text-center"
+          @click="toggleButtons(index + 3)"
```

Comment:
hardcode 3

Reply:
TODO: The section below must be refactored to avoid hard code in v-for